src/main/resources/templates/db_scripts/schema.json

```diff
@@ -519,7 +519,7 @@
       ],
       "ginIndex": [
         {
-          "fieldName": "caption",
+          "fieldName": "displaySummary",
```

Comment:
better to check after enabling the module that RMB deleted index for caption and created for displaySummary

Reply:
![image](https://github.com/folio-org/mod-orders-storage/assets/144257054/2236165c-15ef-4332-bffc-80777b6f281d)

Reply:
It means that old index for caption was deleted and new index for displaySummary was created?

Reply:
Yes, the actual index creation SQL is next to the underlined one.